tests/InstrEngineTests/InstrEngineTests/TestAddInstruction.cs

```diff
@@ -6,12 +6,19 @@
 namespace InstrEngineTests
 {
     [TestClass]
+    // "dotnet test" runs tests in-place (instead of copying them to a deployment
+    // directory). DeploymentItems that copy from a source path to a destination
+    // that is the same location are not necessary and cause file locking issues
+    // when initializing the tests. Only use DeploymentItem for paths with different
+    // destinations for .NET Core test assemblies.
+#if !NETCOREAPP
```

Comment:
Not necessary but it might be cleaner if you create something like

[Conditional("...")]
ConditionalDeploymentItemAttribute : DeploymentItem

Reply:
DeploymentItemAttribute is sealed. It also doesn't implement any type of interface or base class, so I don't think I could create another attribute to mimic the behavior and defer to pre-existing behavior.